packages/@magic-sdk/react-native-expo/README.md

```diff
@@ -0,0 +1,57 @@
+# ✨ Magic Authentication JavaScript SDK
+
+ [![<MagicLabs>](https://circleci.com/gh/magiclabs/magic-js.svg?style=shield)](https://circleci.com/gh/magiclabs/magic-js)
+
+ > Magic empowers developers to protect their users via an innovative, passwordless authentication flow without the UX compromises that burden traditional OAuth implementations.
+ <p align="center">
+   <a href="https://github.com/magiclabs/magic-js/blob/master/packages/@magic-sdk/react-native-expo/LICENSE">License</a> ·
+   <a href="https://github.com/magiclabs/magic-js/blob/master/packages/@magic-sdk/react-native-expo/CHANGELOG.md">Changelog</a> ·
+   <a href="https://github.com/magiclabs/magic-js/blob/master/CONTRIBUTING.md">Contributing Guide</a>
+ </p>
+
+ ## 📖 Documentation
+
```

Comment:
Same here, Shall we make a note to tell developers that this split is in beta, and use with cautious. Expo stable version is `@magic-sdk/react-native@x.x.x`